source/common/network/utility.h

```diff
@@ -91,6 +91,20 @@ class Utility {
   static bool isLoopbackAddress(const Address::Instance& address);
 
   /**
+   * @return a address that represents the IPv4 wildcard address (i.e. "0.0.0.0"). Used
+   * during binding to indicate that incoming connections to any local IPv4 address
+   * are to be received.
+   */
+  static Address::InstancePtr getIpv4AnyAddress();
```

Comment:
There isn't a const problem here that I can see. It returns a copy of the the shared_ptr, and the caller is free to modify that copy of the *ptr*, yet will have no impact on the shared object that others may be using.